conda-recipes/main-library/copy_lib.sh

```diff
@@ -4,7 +4,6 @@
 mkdir -p $PREFIX/lib
 mkdir -p $PREFIX/bin
 mkdir -p $PREFIX/include/sls
```

Comment:
Don't we also need to add the following? We must have missed it in the frame synchonizer PR.
cp build/install/bin/slsFrameSynchronizer $PREFIX/bin/.